### DISCUSSION TITLE
QuerMultiple

### DISCUSSION THREAD

**Discussion by pedro-mileipp** [balta-io/forum#228] [Tira Dúvidas] [answered]
- Boa tarde, prezado! O .Read em  var categories = multi.Read<Category>(); teria qual; função exatamente?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @pedro-mileipp, como vai?
De qual curso/aula seria este código?

**Reply by pedro-mileipp** [reply]
- QuerMultiple de Dapper

**Reply by andrebaltieri** [reply]
Bom dia, @pedro-mileipp, como vai?

Esta aula aqui:
https://balta.io/player/assistir/63c6ef55-44ee-445d-9b53-62e06fc969b8/710a562c-6742-4dfd-856a-53c51934973f

```csharp
var query = "SELECT * FROM [Category]; SELECT * FROM [Course]";

            using (var multi = connection.QueryMultiple(query))
            {
                var categories = multi.Read<Category>();
                var courses = multi.Read<Course>();

                foreach (var item in categories)
                {
                    Console.WriteLine(item.Title);
                }

                foreach (var item in courses)
                {
                    Console.WriteLine(item.Title);
                }
            }
```

No caso, o multi.Read<Course>(); executa a leitura... **Read** em inglês significa ler. É neste momento que ele vai executar a query no banco.

Seria isto?

**Reply by pedro-mileipp** [reply]
Ah sim, obrigado!
